todo_app/templates/index.html

```diff
@@ -7,12 +7,26 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add New To-Do</h2>
+      <form action="{{ url_for('newTodo') }}" method="post">
+        <label for="{{ titleFieldName }}">Title:</label>
+        <input type="text" id="{{ titleFieldName }}" name="{{ titleFieldName }}">
```

Comment:
You've added the back-end check for an empty input, so matching that with a `required` attribute to take advantage of browser's in-built front-end validation would be a nice addition